fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/version/GetLatestVersionDAO.java

```diff
@@ -59,7 +59,7 @@ public GetLatestVersionDAO(String adminSchemaName, String schemaName) {
             ps.setString(2, schemaName);
             ResultSet rs = ps.executeQuery();
             while (rs.next()) {
-                String schema = rs.getString(1);
+                String schema = rs.getString(1).toUpperCase();
```

Comment:
I threw this in as uppercase for consistency

Reply:
Performance note: using WHERE UPPER(my_col) = ... can have performance implications because it often means that you can't use an index (on my_col). But in this case, it's unlikely to have a significant impact, because the row counts are relatively low.

Reply:
good point.